content/md/productivity/sequential-edit_fr.md

```diff
@@ -0,0 +1,47 @@
+---
+id: sequential-edit
+themes: products-super-power, boost-your-productivity
+title: "**Modifier séquentiellement** vos produits"
+popular: false
+ee-only: false
+related: product-mass-actions
+---
+
+# Introduction
+
+La modification séquentielle permet d’afficher les unes après les autres les pages Produit à partir d’une sélection de produits, sans passer par la grille des produits. Cela permet de gagner du temps en permettant la navigation directe entre les pages Produit des produits sélectionnés.
+
+Tout utilisateur disposant d'un accès au produit avec des autorisations de modification peut exécuter une modification séquentielle. Il n’existe aucun droit spécifique pour donner accès à cette fonctionnalité.
+
+
+# Comment faire ?
+
+Pour lancer la modification séquentielle du produit :
+1. Recherchez des produits dans la grille des produits avec les filtres appropriés,
+2. Sélectionnez les produits sur lesquels vous souhaitez effectuer l'opération via l'outil de sélection :
+     - `Tous` pour tous les produits enregistrés,
+     - `Tous les éléments visibles` pour tous les produits visibles sur la page en cours,
+     - `Aucune` pour vider la sélection.
+   ![Menu déroulant Edition séquentielle](Products_DropdownmenuSequentialEdit_fr.png)
+3. Cliquez sur le bouton `Modification séquentielle`,
+   ![image](Products_SequentialEditCTA_fr.png)
+4. La page Produit du premier produit de votre sélection est indiquée par une barre de progression en bas de la page,
+   ![image](Products_SequentialEditProgressBar_fr.png)
+5. Enrichissez votre premier produit et cliquez sur le bouton `Enregistrer et passer au suivant` pour afficher et enrichir le prochain produit de votre sélection,
+   ![image](Products_SequentialEditProgressBarSaveandNext_fr.png)
+6. Lorsque tous les produits de votre sélection sont enrichis, cliquez sur le bouton `Enregistrer et terminer` pour revenir à la grille des produits.
+
+
+La barre de progression indique le nombre de produits qui ont déjà été modifiés par rapport au nombre total de produits sélectionnés. La barre de progression est entourée de deux liens à gauche et à droite.
+
+![image](Products_SequentialEditProgressBar2_fr.png)
+
+Le lien de gauche vous permet de revenir au produit précédent, le lien de droite pour passer au produit suivant.
+
+::: ee
+Les produits avec les droits d'accès en lecture seule seront seulement affichés et ne seront pas modifiables.
+
+<!--
```

Comment:
You can remove the whole block, as for now, we couldn't reproduce this behavior.

Reply:
Done, I removed this part